packages/publications/src/Models/PublicationPage.php

```diff
@@ -23,6 +23,7 @@ class PublicationPage extends Concerns\BaseMarkdownPage
 {
     use ValidatesExistence;
 
+    // Fixme: can we make this private or protected without breaking other stuff?
```

Comment:
There are actually a lot of public properties that can be protected. If you have time, I'd love to get your design thoughts on this issue if you take a look at the code in the core framework package. Mainly thinking about the page classes like `HydePage` and so